scripts/generate-rpc-wrappers.py

```diff
@@ -224,6 +224,17 @@
 
 	('s2c_perform', [], []),
 
+	('set_executable_path', [
+		('buffer', 'const char*', 'uint64_t'),
+		('buffer_size', 'uint64_t')
+	], []),
+
+	('get_executable_path', [
+		('pid', 'int32_t'),
+		('buffer', 'char*', 'uint64_t'),
+		('buffer_size', 'uint64_t')
+	], []),
```

Comment:
Following `vchroot_path`, what you should do here is return the actual length as a return value.
```suggestion
	], [
		('actual_length', 'uint64_t'),
	]),
```
See my suggested change to `GetExecutablePath` for more details.